WooCommerce/Classes/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrder+BottomSheet.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+import Yosemite
+
+extension ProductsSortOrder {
```

Comment:
I wonder if this should be a `private` extension inside the `ProductsSortOrderBottomSheetListSelectorDataSource.swift` file so there are less publicly available properties that may not be very useful anywhere else. The `actionSheetTitle` is probably only useful in there, right?

Reply:
yea this sounds like a good idea - right now there's a test case to assert the cell's title matches the corresponding `actionSheetTitle`. however, as we discussed before I think this type of UI testing where we expose what could have been private isn't worthwhile unless the UI is more complicated. I'll move this to its usage and make it private!

Reply:
updated here: https://github.com/woocommerce/woocommerce-ios/pull/2156/commits/da595f366eadb95601d386039396b32d0c8d5635